src/frontend/src/components/LogIn/index.tsx

```diff
@@ -166,6 +166,24 @@ class LogIn extends React.Component<Props, State> {
                 Forgot your password?
               </Link>
             </div>
+            <div className="tc">
+              <Link
+                to="/add-user"
+                className="link underline hover-blue"
+                href="/"
+              >
+                Go to Add User Component
+              </Link>
+            </div>
+            <div className="tc">
+              <Link
+                to="/edit-user"
+                className="link underline hover-blue"
+                href="/"
+              >
+                Go to Edit User Component
+              </Link>
+            </div>
```

Comment:
Shouldn't the edit user be on the top navigation?

Reply:
Perhaps this is a placeholder? Please see issue #594 for details.